docs/source/jit_language_reference.rst

```diff
@@ -0,0 +1,438 @@
+     - Not Supported
+     - We're interpreting these as real integers instead of imaginary numbers
+   * - `2.5. Operators <https://docs.python.org/3/reference/lexical_analysis.html#operators>`_
+     - Partially Supported
+     - Not supported: <<, >>, :=,
+   * - `2.6. Delimiters <https://docs.python.org/3/reference/lexical_analysis.html#delimiters>`_
+     - Partially Supported
+     - Not supported: **=, <<=, >>=, %=, ^=, @=, &=, //=, % for strings
+   * - `3. Data model <https://docs.python.org/3/reference/datamodel.html#>`_
+     - Not Relevant
+     -
+   * - `3.1. Objects, values and types <https://docs.python.org/3/reference/datamodel.html#objects-values-and-types>`_
+     - Not Relevant
+     - id() not supported, type() upcalls to Python
+   * - `3.2. The standard type hierarchy <https://docs.python.org/3/reference/datamodel.html#the-standard-type-hierarchy>`_
+     - Partially Supported
+     - op == with None works in Python, not supported: NotImplemented, Ellipsis, numbers.Complex, bytes, byte arrays, sets, frozen sets, generators, coroutines, async generators, modules, I/O objects, internal objects, slice objects ( though slicing is supported), classmethod
+   * - `3.3. Special method names <https://docs.python.org/3/reference/datamodel.html#special-method-names>`_
+     - Supported
+     -
+   * - `3.3.1. Basic customization <https://docs.python.org/3/reference/datamodel.html#basic-customization>`_
+     - Partially Supported
+     - Not supported: **new**\ , **del**\ , **bytes**\ , **format**\ , **hash**\ ,
+   * - `3.3.2. Customizing attribute access <https://docs.python.org/3/reference/datamodel.html#customizing-attribute-access>`_
+     - Not Supported
+     -
+   * - `3.3.2.1. Customizing module attribute access <https://docs.python.org/3/reference/datamodel.html#customizing-module-attribute-access>`_
+     - Not Supported
+     -
+   * - `3.3.2.2. Implementing Descriptors <https://docs.python.org/3/reference/datamodel.html#implementing-descriptors>`_
+     - Not Supported
+     -
+   * - `3.3.2.3. Invoking Descriptors <https://docs.python.org/3/reference/datamodel.html#invoking-descriptors>`_
+     - Not Supported
+     -
+   * - `3.3.2.4. **slots** <https://docs.python.org/3/reference/datamodel.html#slots>`_
+     - Not Supported
+     -
+   * - `3.3.2.4.1. Notes on using **_slots**\ * <https://docs.python.org/3/reference/datamodel.html#notes-on-using-slots>`_
+     - Not Supported
+     -
+   * - `3.3.3. Customizing class creation <https://docs.python.org/3/reference/datamodel.html#customizing-class-creation>`_
+     - Not Supported
+     -
+   * - `3.3.3.1. Metaclasses <https://docs.python.org/3/reference/datamodel.html#metaclasses>`_
+     - Not Supported
+     -
+   * - `3.3.3.2. Resolving MRO entries <https://docs.python.org/3/reference/datamodel.html#resolving-mro-entries>`_
+     - Not Supported
+     - No inheritance
```

Comment:
No inheritance is a little misleading, because we do have some amount of support for inheritance within modules